FunctionSignatureMap.php

```diff
@@ -12784,7 +12791,7 @@
 'sqlsrv_server_info' => ['array', 'conn'=>'resource'],
 'sqrt' => ['float', 'number'=>'float'],
 'srand' => ['void', 'seed='=>'int', 'mode='=>'int'],
-'sscanf' => ['mixed', 'str'=>'string', 'format'=>'string', '&...w_vars='=>'string|int|float'],
+'sscanf' => ['mixed', 'str'=>'string', 'format'=>'string', '&...w_vars='=>'string|int|float|null'],
```

Comment:
Again, that seems like it would increase false positives. What's the motiviation for this?

Also, `&...w_vars=` means that this is write-only, and Phan completely ignores the input types.

Are there scanf format strings that would set arguments to null (If all arguments were read successfully and argument counts were correct)?

Reply:
> What's the motiviation for this? Just because it _can_ be `null`.